docs/UserGuide.adoc

```diff
@@ -303,14 +339,20 @@ Examples:
 After adding and managing your flashcards, its time to get your A+!
 This section will teach you how to let Flashy's intelligent scheduling algorithm prompt you the best time to study!
 
+[[showdue]]
 ==== Showing a flashcard : `showdue`
-To start using a flashcard, you have to click on a flashcard first.
+To start using a flashcard, you have to know which things are best to study first.
```

Comment:
maybe "you have to know what your study priorities are"? The current one is too singlish already